### ISSUE TITLE
footer isn't sticky

### ISSUE THREAD

**Issue by Pomax** [mozilla/webmaker.org#1212]
based on https://github.com/mozilla/goggles.webmaker.org/issues/167, the page footer may not be at the bottom of the screen (although it'll be at the bottom of the "page") if the browser window is made tall enough.

**Comment by ScottDowne** [comment]
Either I don't understand, or don't really agree.

So you're saying we want the footer on every page to be position: fixed at the bottom of the page?

**Comment by Pomax** [comment]
no. We want it at the bottom of "the page", or "the browser window", depending on whether the page content overflows or not, respectively. So a window that's 800px high, with a page that has content 1500px hight, will be fine - the footer is properly at the bottom. But a window that's 1600px high, with content that only covers 800px, will have the footer "floating mid page" instead of being located at the bottom of the screen

**Comment by ScottDowne** [comment]
Ah, I see what you mean.

Another option would be to expand the bottom of the footer to the bottom of the window. So instead of a white blob, you just get more blue. Similar to this: https://call.mozilla.org/ cept it's grey not blue.

Not sure if it would be better with an unpredictable amount of space between the last of the content and the footer.

**Comment by Pomax** [comment]
with blue, that ends up looking pretty weird

**Comment by ScottDowne** [comment]
Wait, why does that look weird?

It's a matching blue, to match the blue in this footer https://cloud.githubusercontent.com/assets/1119821/5977608/510f705c-a867-11e4-8709-b5fc1082a156.png

That's an example. It's whatever colour matches the footer, not blue as a rule.

**Comment by Pomax** [comment]
yeah I mean it *looks* weird, since it's then a giant block of blue
